src/expression.c

```diff
@@ -947,7 +947,7 @@ Expression *resolveUFCSProperties(Scope *sc, Expression *e1, Expression *e2 = NU
  * Perform semantic() on an array of Expressions.
  */
 
-bool arrayExpressionSemantic(Expressions *exps, Scope *sc)
+bool arrayExpressionSemantic(Expressions *exps, Scope *sc, bool preserveErrors)
```

Comment:
That would be fine by me. Although I would prefer to have some input on this from Walter and/or yebblies. Also, I don't think I will make any more changes unless I can get some more assurances that this will actually get merged at some point.